<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditProxyDialogClass</class>
 <widget class="QDialog" name="EditProxyDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proxy Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
     <item>
      <widget class="QListWidget" name="ltwProxyList"/>
     </item>
     <item>
      <widget class="QWidget" name="wdtProperties" native="true">
       <layout class="QFormLayout" name="formLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="lblName">
          <property name="text">
           <string>Name:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="buddy">
           <cstring>lneName</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lneName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblType">
          <property name="text">
           <string>Type:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="buddy">
           <cstring>cmbType</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cmbType"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblHost">
          <property name="text">
           <string>Host:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="buddy">
           <cstring>lneHost</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lneHost"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblPort">
          <property name="text">
           <string>Port:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="buddy">
           <cstring>spbPort</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spbPort">
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lblUser">
          <property name="text">
           <string>User:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="buddy">
           <cstring>lneUser</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lneUser"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="lblPassword">
          <property name="text">
           <string>Password:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="buddy">
           <cstring>lnePassword</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lnePassword">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblDefault">
       <property name="text">
        <string>Default Proxy:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="buddy">
        <cstring>cmbDefault</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbDefault">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbtAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="btbButtons">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ltwProxyList</tabstop>
  <tabstop>lneName</tabstop>
  <tabstop>cmbType</tabstop>
  <tabstop>lneHost</tabstop>
  <tabstop>spbPort</tabstop>
  <tabstop>lneUser</tabstop>
  <tabstop>lnePassword</tabstop>
  <tabstop>cmbDefault</tabstop>
  <tabstop>pbtAdd</tabstop>
  <tabstop>pbtDelete</tabstop>
  <tabstop>btbButtons</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
